docs/src/ref_impl/installer.md

```diff
@@ -0,0 +1,73 @@
+<!--
+    Copyright 2022-2023 TII (SSRC) and the Ghaf contributors
+    SPDX-License-Identifier: CC-BY-SA-4.0
+-->
+
+# Installer
+
+## Configuring and Building Installer for Ghaf
+
+You can obtain the installation image for your Ghaf configuration. To check possible configuration options, see [Modules Options](../ref_impl/modules_options.md#ghafinstallerenable).
+
+1. Set `ghaf.installer.enable` to `true`.
+2. Add nixos-generators module to `ghaf.installer.imgModules` list to configure installer image type.
+3. Choose installer modules from `ghaf.installer.installerModules` and set `ghaf.installer.enabledModules` to list of their names.
+4. Write code for the installer in `ghaf.installer.installerCode`.
+
+Then you can build an installer image using the command:
```

Comment:
Is this extra?

Reply:
Good catch. Should be above line 44, not here.